rules/windows/sysmon/sysmon_dcom_iertutil_dll_hijack.yml

```diff
@@ -23,7 +23,7 @@ detection:
         EventID: 7
         Image|endswith: '\Internet Explorer\iexplore.exe'
         ImageLoaded|endswith: '\Internet Explorer\iertutil.dll'
-    condition: selection_one or selection_two
+    condition: 1 of selection_*
```

Comment:
This rule needs to be split into 2 rules. For `file_event` and `image_load`

Reply:
Also, there is another rule that should be split. Take a look at `sysmon_accessing_winapi_in_powershell_credentials_dumping.yml`